lib/plugins/json_body_parser.js

```diff
@@ -25,7 +25,7 @@ function jsonBodyParser(options) {
     var override = options.overrideParams;
 
     function parseJson(req, res, next) {
-        if (req.getContentType() !== 'application/json' || !req.body) {
```

Comment:
Actually, this needs to be fixed in request.js, such that it applies to all content-types. I'll do that. Thanks for the PR though.